contracts/misc/AutoCompoundApe.sol

```diff
@@ -29,6 +30,7 @@ contract AutoCompoundApe is
     ApeCoinStaking public immutable apeStaking;
     IERC20 public immutable apeCoin;
     uint256 public bufferBalance;
+    uint256 public bufferStakingBalance;
```

Comment:
```suggestion
    uint256 public stakingBalance;
```
It's not a buffer that will be cleared. So maybe we use stakingaBalance or stakingAmount directly?

Reply:
ok